### ISSUE TITLE
Is it possible to encrypt text strings in Python using sops?

### ISSUE THREAD

**Issue by AndreyRojasD** [getsops/sops#1380]
For instance, my use case is as follows.

```py
import sops

def encrypt(password):
    return sops.encrypt(password)
```

**Comment by felixfontein** [comment]
The Python version of sops is deprecated and no longer supported. For version 2.0.0 sops was rewritten in go. If you want to encrypt something with sops from Python, use https://docs.python.org/3/library/subprocess.html to call the sops binary.

**Comment by silentpete** [comment]
I agree with @felixfontein and have implemented it in a program to verify it works as expected with the new Go based `sops` package.
Using Python 3.12 and SOPS v3.8.1

**Comment by nipil** [comment]
You can use SOPS Y python module : 

https://github.com/nikaro/sopsy
